site/src/App/Code/Code.tsx

```diff
@@ -1,16 +1,39 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, ReactChild } from 'react';
 import { useStyles } from 'react-treat';
 import copy from 'copy-to-clipboard';
+import memoize from 'lodash/memoize';
+import reactElementToJSXString from 'react-element-to-jsx-string';
+import prettier from 'prettier/standalone';
+import babylonParser from 'prettier/parser-babylon';
 import { createUrl } from 'sku/playroom/utils';
 import classnames from 'classnames';
 import { useConfig } from '../ConfigContext';
-import { Box, Text } from '../../../../lib/components';
+import { Box, Stack, Text, BraidProvider } from '../../../../lib/components';
+import { seekAnz } from '../../../../lib/themes';
 import { BoxProps } from '../../../../lib/components/Box/Box';
 import { FieldOverlay } from '../../../../lib/components/private/FieldOverlay/FieldOverlay';
 import { CopyIcon } from './CopyIcon';
 import { PlayIcon } from './PlayIcon';
 import * as styleRefs from './Code.treat';
 
+// @ts-ignore
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import editorTheme from './editorTheme';
+
+const formatSnippet = memoize(
+  snippet =>
+    prettier
+      .format(snippet, {
+        parser: 'babylon',
```

Comment:
Im prettier sure this has been renamed to `babel`, from memory it throws a warning as being deprecated.